pkg/oc/cli/admin/migrate/migrator.go

```diff
@@ -594,12 +594,14 @@ type migrateTracker struct {
 func (t *migrateTracker) report(prefix string, info *resource.Info, err error) {
 	ns := info.Namespace
 	if len(ns) > 0 {
-		ns = "-n " + ns
+		ns = " -n " + ns
 	}
+	groupResource := info.Mapping.Resource.GroupResource()
+	groupResourceStr := (&groupResource).String()
```

Comment:
is the string method only defined on a pointer receiver?

Reply:
Yup